setup.py

```diff
@@ -66,9 +74,129 @@
 }
 requires['full'] = sum(list(requires.values()), [])
 
+
+class BuildManPage(Command):
```

Comment:
Done. I kept the custom build_py class though. This is the only place where you can see that build_py is not just build_py -- I consider this declarative.